### ISSUE TITLE
Warn people about where this action will push to if not specified

### ISSUE THREAD

**Issue by joelcoxokc** [actions-js/push#24]
Please let people know that even if you run this workflow under a workflow_dispatch on a dev branch, if `branch` is not specified then it will push all dev code to your main branch. 

We have four branches. 
main -> staging -> qa -> dev

I ran this action on my qa branch... and did not specify the `branch` property. And it pushed all my dev code to our main production branch. 

This could be very dangerous in many environments. 

I personally think that the `branch` should not default to main, but should default to the branch that HEAD is on

**Comment by lpender** [comment]
I agree -- I think it should push to whatever branch it's in by default.

**Comment by ImanMahmoudinasab** [comment]
@devenes I think the default value for the branch should change to the current branch unless the user defines the branch name. I can apply this change as part of the next breaking change.
